region/norfair/east/Spiky Acid Snakes Tunnel.json

```diff
@@ -339,6 +339,14 @@
         "Delay the damage boost from the spikes slightly in order to rise above the lava before moving."
       ]
     },
+    {
+      "link": [1, 2],
+      "name": "Ceiling Bomb Jump Over Spikes",
+      "requires": [
+        "h_heatProof",
+        "canLongCeilingBombJump"
```

Comment:
This has a sloped ceiling on both sides, so it would take diagonal bomb jumping into returning to neutral and staggered IBJ before the longCIBJ

Reply:
Maddo was testing ceiling bomb jump on gentle up slopes and horizontal ceilings and said they felt the same. He didn't try for long, and never finished crossing the room, so they could be a bit different: https://videos.maprando.com/video/1246

I then tested it on gentle down, and got it to work for about 2 tiles before I would fail, so I know it's doable there, but I have no idea how it compares to horizontal. Instead, it looks like an unmorph reset fall speed works here. 

I didn't think DIBJ into IBJ was something we wanted to include, or there would be a couple more uses (i.e. R to L Crateria Supers)

Reply:
I tried this out, and you can CIBJ up gentle slopes, so I learned something.  I had too much trouble getting a CIBJ going for the downslope but you can reach the ledge with an unmorph like you said.

DIBJ into IBJ looks like you could get good enough to react to getting the right pixel, but I couldn't read that well enough.  So it'd be like longCIBJ tier and not a priority.

Reply:
And I don't see `1246` as linking to a video

Reply:
Ah, looks like it's listed as a disabled video, since the CIBJ failed part was through and fell. With that link you can then filter to disabled videos and it will show.